Controller/RestApi.php

```diff
@@ -45,38 +45,55 @@ public function list() {
   }
 
   /**
-   * Run schedulling.
+   * Run scheduling.
    */
   public function run() {
     $results = $this->list();
     $processed = [];
-    $collections = [];
+    $data = [];
     $entries = [];
-    foreach ($results['active'] as $data) {
-      $type = $data['schedule']['type'];
-      if (!isset($collections[$data['_collection']])) {
-        $collection = $this->app->module('collections')->collection($data['_collection']);
-        $collections[$data['_collection']] = $collection;
+    foreach ($results['active'] as $result) {
```

Comment:
probably could we decouple better singletons from collections functionality, and reduce the size of the run function